basic-stashing/README.md

```diff
@@ -0,0 +1,42 @@
+# Git Kata: Basic stashing 
+
+## Setup:
+In your terminal, run
+```
+$ . ./setup.sh
+```
+
+## The task
+You are working on your project.  you've staged some work and have some unstaged work as well.

```

Comment:
markdown best practice is to have new line after headers ( Sorry for the nitpickery! )